packages/web3-eth/src/web3_eth.ts

```diff
@@ -66,14 +57,14 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI> {
 
 	public async getStorageAt(
 		address: Address,
-		storageSlot: Uint256,
+		storageSlot: Numbers,
 		blockNumber: BlockNumberOrTag = this.defaultBlock,
 	) {
-		return rpcMethods.getStorageAt(this.requestManager, address, storageSlot, blockNumber);
+		return rpcMethodsWrappers.getStorageAt(this, address, storageSlot, blockNumber);
```

Comment:
Input formatting is only done by `rpc_method_wrappers.ts`, so we must change what function we're calling